main.go

```diff
@@ -33,7 +35,9 @@ func main() {
 		}
 	}
 
+	conjurVersion := os.Getenv("CONJUR_VERSION")
```

Comment:
Can you add this to the config options in the readme? I can add it to dockerhub after that.

Also, should this have a default? To "5"?